### ISSUE TITLE
AsciiDocPreviewSettings in asciidoc.xml have no effect

### ISSUE THREAD

**Issue by thorstenkampe** [asciidoctor/asciidoctor-intellij-plugin#458]
Setting `toc` and `sectnums`  to `false` in application.xml has no effect. These are not configurable via the GUI.
```xml
<application>
  <component name="AsciidocApplicationSettings">
    <AsciiDocPreviewSettings DisabledInjectionsByLanguage="" KrokiUrl="" EnabledFoldedAttributeReferencedExperimental="true">
      <attribute key="experimental" value="true" />
      <attribute key="icons" value="font" />
      <attribute key="sectnums" value="false" />
      <attribute key="toc" value="false" />
    </AsciiDocPreviewSettings>
  </component>
</application>
```

**Comment by ahus1** [comment]
I assume that you want to set attributes for the HTML preview of the plugin.

The settings in the XML are supposed to be configured by the settings UI, see below. This works for example with 2019.x. For 2020.1 it seems to be broken for attributes, I'll fix that in the next pre-release.

At the moment you can only set attributes via the config, you can't unset them. 

As `sectnums` and `toc` are unset by default, I suggest you remove them from the configuration.

Please note: If the document you view re-defines these attributes, the settings from the document have priority and will override the settings of the IDE.

If you are interested in configuring project-specific defaults for the preview, please consider adding a _.asciidoctorconfig_ file to your project, see here: https://github.com/asciidoctor/asciidoctor-intellij-plugin/wiki/Support-project-specific-configurations. This gives you full control over unsetting attributes and folder specific attributes. Still, if a document overrides attributes, these will have priority.

Please let me know if this solved your problem. If not, please provide a little bit more context or an example so I can find a solution.

![image](https://user-images.githubusercontent.com/3957921/79042887-b3b1c080-7bfb-11ea-882f-bc6ea618a7f4.png)

**Comment by ahus1** [comment]
Pre-Release 0.30.57 is available that fixes the settings UI for IntelliJ 2020.1+

The new pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).

**Comment by thorstenkampe** [comment]
The real issue was the broken configuration through the GUI. This works fine now. Thanks.
